.github/linters/.markdown-lint.yml

```diff
@@ -0,0 +1,12 @@
+MD007: false
```

Comment:
can we add comments on what each rule is?

Reply:
I added three comments at the top of this markdownlint config file.

I think it's best if we just link to the offical rules.

Reply:
Hey @rberrelleza also when you run `markdownlint` on the command line it prints out the line numbers with the failing issues.

This is an image from another repo:

<img width="1194" alt="Screen Shot 2021-04-13 at 8 12 20 pm" src="https://user-images.githubusercontent.com/418747/114536831-d1918580-9c94-11eb-908f-b6aa282a75f6.png">